src/VirtoCommerce.CatalogModule.Data/Search/Indexing/AggregationConverter.cs

```diff
@@ -235,11 +236,16 @@ protected virtual IList<AggregationItem> GetAttributeAggregationItems(IList<Aggr
         {
             var result = aggregationResponseValues
                 .Select(v =>
-                    new AggregationItem
+                {
+                    return new AggregationItem
                     {
-                        Value = v.Id,
+                        Value = v.Id.Contains(ModuleConstants.OutlineDelimiter) ? v.Id.Split(ModuleConstants.OutlineDelimiter)[0] : v.Id,
                         Count = (int)v.Count,
-                    })
+                        Labels = v.Id.Contains(ModuleConstants.OutlineDelimiter) ?
+                                    new[] { new AggregationLabel { Label = v.Id.Split(ModuleConstants.OutlineDelimiter)[1] } } :
```

Comment:
Probably you need to check the length to avoid possible IndexOutOfReange exception